### PR TITLE
DB-10125: Installation notes for DB-9578 for CDH put changes into wrong site.xml file

### PR DESCRIPTION
**Pull request by ipraznik-splice**

### CODE REVIEW

**Comment by OlegMazurov** [comment]
_dfs.client.*_ settings in _hdfs-site.xml_ affect all HDFS client in the cluster. When in _hbase-site.xml_, they affect only HBase/Splice. Moreover, when HBase sets HDFS client configuration parameters in its _hbase-default.xml_ (embedded in _hbase-common.jar_) they override those set in _hdfs-site.xml_ (see f.e. _dfs.client.read.shortcircuit_).
I believe the original instructions were correct.

File: platforms/cdh6.3.0/docs/CDH-installation.md [hunk 1]
```diff
@@ -171,7 +171,15 @@ use your data disks.
     </tbody>
    </table>
 
-3. Click the `Save Changes` button.
+3. Set the value of `HDFS Client Advanced Configuration Snippet (Safety Valve) for hdfs-site.xml`:
+   ````
+   <property><name>dfs.client.block.write.retries</name><value>6</value></property>
+   <property><name>dfs.client.block.write.locateFollowingBlock.retries</name><value>10</value></property>
+   <property><name>dfs.client.block.write.replace-datanode-on-failure.policy</name><value>ALWAYS</value></property>
+   <property><name>dfs.namenode.replication.min</name><value>2</value></property>
```
Comment (dgomezferro) [inline platforms/cdh6.3.0/docs/CDH-installation.md:179]: This last property has to go into the hdfs-site.xml for the service section. As discussed in https://splicemachine.atlassian.net/browse/DB-10125 it seems that the remaining properties have to go in hbase-site.xml as they were already.

**Review by dgomezferro — CHANGES_REQUESTED** [review]
See discussion in https://splicemachine.atlassian.net/browse/DB-10125 it seems most changes aren't needed. Sorry for the confusion!

File: platforms/cdh6.3.0/docs/CDH-installation.md (hunk 1 above)
Comment (ipraznik-splice) [reply platforms/cdh6.3.0/docs/CDH-installation.md]: I moved this property "dfs.namenode.replication.min" to the part "2. Change the values of these settings:"
        <tr>
            <td><code>Minimal Block Replication</code></td>
            <td><code>2</code></td>
        </tr>
is it right place? Checked in Cloudera 6.3.0 HDFS Configuration for the name.

Comment (ipraznik-splice) [reply platforms/cdh6.3.0/docs/CDH-installation.md]: dfs.namenode.replication.min go into 
Configure HDFS
2. Change the values of these settings:
        <tr>
            <td><code>Minimal Block Replication</code></td>
            <td><code>2</code></td>
        </tr>

dfs.client.* are moved back into hbase-site.xml

**Review by jyuanca — APPROVED** [review]

File: platforms/cdh6.3.0/docs/CDH-installation.md (hunk 1 above)
Comment (dgomezferro) [reply platforms/cdh6.3.0/docs/CDH-installation.md]: @ipraznik-splice it appears HBase always overrides `dfs.namenode.replication.min`, so if we want it to apply it needs to be set in hbase-site.xml.
But we should hold off on that until @OlegMazurov completes some latency tests to check whether this change has a big impact in performance.

**Comment by jyuanca** [comment]
@dgomezferro @OlegMazurov Can you please take another look at this PR?

File: platforms/cdh6.3.0/docs/CDH-installation.md (hunk 1 above)
Comment (carolp-503) [reply platforms/cdh6.3.0/docs/CDH-installation.md]: Oleg has completed performance tests which did not show a significant performance degradation.

**Review by dgomezferro — APPROVED** [review]

**Pull request merged**